src/bitfield.cpp

```diff
@@ -59,7 +59,7 @@ namespace libtorrent
 	{
 		int ret = 0;
 		const int words = num_words();
-#if TORRENT_HAS_SSE
+#if (TORRENT_HAS_SSE && defined _MSC_VER && _MSC_VER >= 1600)
```

Comment:
oh, sorry, i forgot it !
I only thought about vs.

bug only make TORRENT_HAS_SSE be set to false for old versions of msvc is not enough, because _mm_crc32_u32 is not defined。